ADOPTERS.md

```diff
@@ -3,8 +3,8 @@
 _If you're using ORT in your organization, please add your organization's name to the alphabetically sorted table below.
 This really helps the project to gain momentum and credibility. It's a small contribution back to the project with a big impact._
 
-| Organization | Contact | Description of Use |
-|--------------|---------|--------------------|
-| [Alliander](https://www.alliander.com/en/open-source/) | ospo@alliander.com | Alliander is mainly interested in the security side of the software supply chain. Alliander uses multiple tools and has evaluated even more, but currently ORT is unbeaten. ORT provides acurate and destailed scan results and allows us to define a through policy to use in evaluation. We are currently working to adopt ORT for thorough scans in addition to other tools. We're working with the community to get ORT running in GitHub Actions. |
-| [Deutsche Telekom AG](https://www.telekom.de/) | daniel.eder@magenta.at | Deutsche Telekom is in the process of adopting ORT as part of our CI/CD chain and is working on a custom ORT reporter in our [ORT fork](https://github.com/telekom/ort). |
-| [Sovereign Cloud Stack](https://scs.community/) | scs@osb-alliance.com | Currently in the process of evaluating how to generate a complete SBOM of our stack and secure the supply chain. We plan to use ORT to analyze our reference implementation as well as to add ORT to our [recommended operator's toolset](https://scs.community/about/#technological-vision). |
+| Organization | Contact | Description of Use                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
This also changes quite a bit of formatting. Could you please move the formatting changes, if intended, to another commit?

Reply:
This was unintended. Hope it's now fixed..